djstripe/stripe_objects.py

```diff
@@ -941,8 +976,17 @@ def str_parts(self):
 # ============================================================================ #
 
 class StripeAccount(StripeObject):
+    """Repr of Stripe Account object.
```

Comment:
put this as "coming soon as discussed earlier" and let the docstring be inherited in models.py